src/objects/bio/bio.scss

```diff
@@ -14,18 +14,27 @@
     'avatar'
     'content'
     'meta';
-  grid-template-rows: repeat(3, minmax(0, auto));
 
   @media (min-width: breakpoint.$l) {
+    gap: 1em;
     grid-template-areas:
       'avatar content content'
       'meta content content';
     grid-template-columns: repeat(3, 1fr);
   }
 }
 
+///
+/// 1. Match prototype designs
+///
+
 .o-bio__avatar {
   grid-area: avatar;
+  inline-size: 12em; // 1
```

Comment:
Should this be a `max-width` (or `max-inline-size`? Haven't actually written that before...) to promote fluidity at small sizes?

Also, can we introduce a new `size.$square-avatar-large` token for this value? See: https://deploy-preview-1747--cloudfour-patterns.netlify.app/?path=/story/design-tokens-size--page#sizing